PowerUp/app/src/main/java/powerup/systers/com/MainActivity.java

```diff
@@ -30,6 +30,14 @@ public void onClick(View v) {
 				startActivityForResult(myIntent, 0);
 			}
 		});
+		ImageButton about = (ImageButton)findViewById(R.id.aboutButtonMain);
+		about.setOnClickListener(new View.OnClickListener() {
+			@Override
+			public void onClick(View v) {
+				Intent myIntent = new Intent(MainActivity.this,AboutActivity.class);
```

Comment:
Put spaces between variables when you call a function. 
`Intent(MainActivity.this, AboutActivity.class);`
If you go to Code->Format code you will also get your code correctly formated